webapp/package.json

```diff
@@ -37,6 +37,7 @@
     "react": ">=16.3.0",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
+    "react-hook-thunk-reducer": "^0.2.4",
     "react-icons": "^3.11.0",
     "react-router-dom": "^5.2.0",
     "regenerator-runtime": "^0.13.5"
```

Comment:
require('string_decoder') this is missing from package.json, i encountered a build error